test/sample_set_filter_test.py

```diff
@@ -220,3 +220,9 @@ def test_order_of_ops(self):
             }
         ])
         self.assertEqual(len(samples_merge), 206)
+
+    def test_get_sampleset_meta(self):
+        ret = self.serviceImpl.get_sampleset_meta({'sample_set_ref': self.sample_set_2_ref})
```

Comment:
missing `ctx` argument. Also I think `sample_set_ref` should be a list argument (e.g. [self.sample_set_2_ref])